packages/backend/src/graphql/directives/isAdmin.test.ts

```diff
@@ -0,0 +1,137 @@
+import { ApolloServer } from 'apollo-server'
+import { makeExecutableSchema } from 'graphql-tools'
+import { isAdminDirectiveTransformer } from './isAdmin'
+import {
+  ApolloClient,
+  NormalizedCacheObject,
+  InMemoryCache,
+  createHttpLink,
+  gql
+} from '@apollo/client'
+import fetch from 'cross-fetch'
+
+describe('isAdmin Directive', (): void => {
+  let server: ApolloServer
+  let client: ApolloClient<NormalizedCacheObject>
+
+  beforeAll(
+    async (): Promise<void> => {
+      const schema = makeExecutableSchema({
+        typeDefs: [
+          `
+        type Query {
+          hello: QueryResponse @isAdmin
+        }
+
+        type QueryResponse {
+          code: String!
+          success: Boolean!
+          message: String!
+        }
+
+        directive @isAdmin on OBJECT | FIELD_DEFINITION
+      `
+        ],
+        resolvers: {
+          Query: {
+            hello: () => {
+              return {
+                code: '200',
+                success: true,
+                message: 'Hello World!'
+              }
+            }
+          }
+        }
+      })
+
+      const schemaWithDirectives = isAdminDirectiveTransformer(schema)
+
+      server = new ApolloServer({
+        schema: schemaWithDirectives,
+        context: ({ req }): { admin: boolean } => {
+          return {
+            admin: req.headers['x-api-key'] == 'qwertyuiop1234567890'
+          }
+        }
+      })
+      await server.listen(3010)
+      const httpLink = createHttpLink({
+        uri: 'http://localhost:3010/graphql',
```

Comment:
I got a random `Error: listen EADDRINUSE: address already in use :::3010` one time while running the tests.
Can this be changed to a different port than the auth directirve test's server?